lib/ladybug/energy_model/room.rb

```diff
@@ -121,7 +119,86 @@ def create_openstudio_object(openstudio_model)
+            people_object = people.get
+            people_object.setSpace(openstudio_space)
+          end
+        end
+
+        if @hash[:properties][:energy][:lighting]
+          lighting_object = nil
+          lighting = openstudio_model.getLightsByName(@hash[:properties][:energy][:lighting][:name])
+          unless lighting.empty?
+            lighting_object = lighting.get
+          end
+          lighting_object.setSpace(openstudio_space)
+        end
+
+        if @hash[:properties][:energy][:electrical_equipment]
+          electrical_equipment_object = nil
+          electrical_equipment = openstudio_model.getElectricEquipmentByName(@hash[:properties][:energy][:electrical_equipment][:name])
+          unless electrical_equipment.empty?
+            electrical_equipment_object = electrical_equipment.get
+          end
+          electrical_equipment_object.setSpace(openstudio_space)
+        end
+        
+        if @hash[:properties][:energy][:gas_equipment]
+          gas_equipment_object = nil
+          gas_equipment = openstudio_model.getGasEquipmentByName(@hash[:properties][:energy][:gas_equipment][:name])
+          unless gas_equipment.empty?
+            gas_equipment_object = gas_equipment.get
+          end
+          gas_equipment_object.setSpace(openstudio_space)
+        end
+
+        if @hash[:properties][:energy][:infiltration]
+          infiltration_object = nil
+          infiltration = openstudio_model.getSpaceInfiltrationDesignFlowRate(@hash[:properties][:energy][:infiltration][:name])
+          unless infiltration_object.empty?
+            infiltration_object = infiltration.get
+          end
+          infiltration_object.setSpace(openstudio_space)
+        end
+          
+        if @hash[:properties][:energy][:ventilation] 
+          ventilation_object = nil
+          ventilation = openstudio_model.getDesignSpecificationOutdoorAirByName(@hash[:properties][:energy][:ventilation][:name])
+          unless ventilation_object.empty?
+            ventilation_object = ventilation.get
+          end
+          ventilation_object.setSpace(openstudio_space)
+        end
+
+        #if @hash[:properties][:energy][:setpoint]
+        #  setpoint_thermostat_object = nil
+        #  setpoint_thermostat = openstudio_model.getThermostatSetpointDualSetpointByName(@hash[:properties][:energy][:setpoint])
+        #  unless setpoint_thermostat_object.empty?
+        #    setpoint_thermostat_object = setpoint_thermostat.get
+        #  end
+        #end
+
+        #if @hash[:properties][:energy][:setpoint][:humidification_schedule] or @hash[:properties][:energy][:setpoint][:dehumidification_schedule]
+        #  setpoint_humidistat_object = nil
+        #  setpoint_humidistat = openstudio_model.getZoneControlHumidistatByName(@hash[:properties][:energy][:setpoint])
+        #  unless setpoint_humidistat_object.empty?
+        #    setpoint_humidistat_object = setpoint_humidistat.get
+        #  end
+        #end
```

Comment:
Is this code commented out here because you plan to send a separate PR that adds a thermal zone to each space?  Or are you still planning to address it as part of this PR?

Reply:
If it's the former, putting a `TODO` comment before all of these comments is good practice.